src/schemas/HelmValuesGitRepositoryRequest.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
```

Comment:
Shouldn't it be null ?

Reply:
My understanding is that when used in a 'oneof,' it should not be nullable. Am I wrong?